src/Interpreters/Context.cpp

```diff
@@ -528,6 +528,9 @@ struct ContextSharedPart : boost::noncopyable
     /// Only for system.server_settings, actually value stored in reloader itself
     std::atomic_size_t config_reload_interval_ms = ConfigReloader::DEFAULT_RELOAD_INTERVAL.count();
 
+    double min_os_cpu_wait_time_ratio_to_drop_connection = 15.0;
```

Comment:
Why don't use the same cakes as for the settings?

Reply:
I'm not sure I understood it right, but if you mean why not use the same values as from the `Settings.cpp`, then the answer is those are different settings. `min_os_cpu_wait_time_ratio_to_drop_connection` controls connection drops, while `min_os_cpu_wait_time_ratio_to_throw` controls query rejection, and those should have different values

Reply:
Oh, right I missed it